addons/sale/models/sale_order.py

```diff
@@ -257,12 +257,6 @@ def _rec_names_search(self):
     medium_id = fields.Many2one(ondelete='set null')
     source_id = fields.Many2one(ondelete='set null')
 
-    # Followup ?
```

Comment:
This comment can be kept, even if there is only the tags left in that 'section'

Reply:
done